requirements/base.txt

```diff
@@ -16,3 +16,5 @@ python-dateutil==2.8.2
 python-magic==0.4.27
 rules==3.3
 XlsxWriter==3.1.2
+celery==5.3.1
+redis==5.0.0
```

Comment:
we might not need redis here as a4 doesn't use it / is transcient to which celery broker is used. The actual celery config will be in a+/mb, so maybe we just add it there?

Reply:
done